core/test/fixtures/fraggle-rock/artifacts/sample-flow-artifacts.json

```diff
@@ -295067,7 +295067,8 @@
         "Doctype": {
           "name": "html",
           "publicId": "",
-          "systemId": ""
+          "systemId": "",
+          "documentCompatMode": "CSS1Compat"
```

Comment:
This is only for one flow step, shouldn't there be more

Reply:
I suppose! The update script didn't work so I just updated one thing 🙈